pkg/config/repoconfig/repoconfig.go

```diff
@@ -0,0 +1,75 @@
+package repoconfig
+
+import (
+	"fmt"
+
+	"github.com/submariner-io/pr-brancher-webhook/pkg/git"
+	"gopkg.in/yaml.v2"
+	"k8s.io/klog"
+)
+
+const (
+	defaultApprovals = 2
+	defaultLabel     = "ready-to-test"
+)
+
+type BotConfig struct {
+	LabelApproved *struct {
+		Approvals *int
+		Label     *string
+	} `yaml:"label-approved"`
+}
+
+type Source struct {
+	SSHURL string
+	RepoName string
+	Remote string
+	SHA string
+}
+
+func Read(src *Source) (*BotConfig, error) {
+	gitRepo, err := git.New(src.RepoName, src.SSHURL)
+	if err != nil {
+		return nil, err
+	}
+
+	gitRepo.Lock()
+	defer gitRepo.Unlock()
+
+	err = gitRepo.EnsureRemote(src.Remote, src.SSHURL)
```

Comment:
```suggestion
	err = gitRepo.EnsureRemote("origin", src.SSHURL)
```

It's constant for both, no need to pass it as a variable right now (in the future it can be refactored if necessary)

Reply:
Done